docs/contributing-to-airbyte/developing-on-docker.md

```diff
@@ -0,0 +1,46 @@
+# Developing on docker
```

Comment:
This doc needs to be added to SUMMARY.md so it actually shows up.

Reply:
Done